packages/elastic-handler/src/elasticHandler.ts

```diff
@@ -94,7 +86,11 @@ export class ElasticHandler<
   }: ReadonlyRecursive<LogEntry<TTaskContext, TGlobalContext>>): Record<string, any> {
     return {
       ...entry,
-      context: clone({ ...(taskContext || {}), ...globalContext, subtaskIds: this.asyncStorage.getStore() }),
+      context: {
+        ...globalContext,
+        ...(taskContext || {}),
+        subtaskIds: this.asyncStorage.getStore(),
```

Comment:
tohle budu muset prepsat. Jak jsi odebral taskId z loggeru

Reply:
spíš jen přejmenovat na něco jako `taskIds` nebo `taskStack` - ono to celou dobu je nezávislý, tak jak jsi to napsal, akorát když jsi tam měl to "globální" `taskId`, tak ten top-level task měl efektivně dva identifikátory - globální a první v tomhle seznamu - který byly ale různý, což moc nedávalo smysl

Reply:
Prepis stejne nejaky bude az to budu zavadet do Tetrisu, ale jeste nevim na co vsechno prijdu, tak bych to udelal s tim.